### DISCUSSION TITLE
Deepgram Beta Websocket TTS not working

### DISCUSSION THREAD

**Discussion by sushangautam** [deepgram/community#911] [General help] [answered]
We were using this endpoint for TTS service of Deepgram. It was working well, suddenly from today we are getting this error : (503 service unavaialable)

wss://api.beta.deepgram.com/v1/speak?encoding=mulaw&sample_rate=8000&container=none

Is there any changes to this beta version or is it removed or released to production ??

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by deepgram-community** [comment, marked as answer]
Hi sushangautam! At this point you should move off our beta endpoint for TTS and use our production endpoint for REST based TTS requests >   `http://api.deepgram.com|api.deepgram.com` . If you happen to be testing out our new web socket functional for TTS you can still use this beta endpoint. The beta endpoint did have a small outage yesterday which has since been resolved.
> Naomi
